src/QsCompiler/Compiler/ExternalRewriteSteps.cs

```diff
@@ -17,50 +17,88 @@ namespace Microsoft.Quantum.QsCompiler
 {
     internal static class RewriteSteps
     {
-        /// <summary>
-        /// Concrete implementation of the rewrite steps options such that the configured options may be null. 
-        /// </summary>
-        internal class RewriteStepOptions : IRewriteStepOptions
```

Comment:
Did you get rid of IRewriteStepOptions to simplify the rewrite interface? Are there no more options to consider other than output folder?